test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -15,7 +15,9 @@ import (
 	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
 )
 
+//TODO get commonly used variables from one shared file that defines constants
```

Comment:
Do we want to leave this TODO ?

Reply:
yes, this will be an enhancement for the tests